Installation/release.sh

```diff
@@ -49,7 +49,45 @@ else
     echo "gitbook missing from your system"
     exit 1
 fi
+if markdown --version; then
+    echo "discount / markdown found."
+else
+    echo "markdown utility that should be shipped via the discount package is missing on your system"
+    exit 1
+fi
+                           
+if html2text -version; then
+    echo "html2text found."
+else
+    echo "html2text missing from your system"
+fi
```

Comment:
Why does this not exit?

Reply:
because of the user didn't install it? or its not in the path? However, the release process requires it, so we have to abort.